### DISCUSSION TITLE
DHCP CONFIGURATION

### DISCUSSION THREAD

**Discussion by loictlpk** [pfsensible/core#80] [Q&A]
Hi all, I'm trying to setup dhcp but it's not working... why ? 

- name: interfaces assignements 1
  pfsensible.core.pfsense_interface:
    descr: LAN2
    interface: xn2
    ipv4_type: static  # dhcp, static or none
    ipv4_address: "10.0.0.1"
    ipv4_prefixlen: 24
    enable: True

- name: interfaces assignements 2 for DHCP
  pfsensible.core.pfsense_interface:
    descr: LAN21
    interface: xn3
    ipv4_type: dhcp  
    enable: True

- name: setup DHCP
  pfsensible.core.pfsense_dhcp_static:
    name: "DHCP"
    macaddr: "52:93:81:a5:2e:1d"
    netif: LAN2
    ipaddr: "10.0.0.1"
    state: present

**Comment by opoplawski** [comment]
What output do you get when run in verbose mode?
